turbopack/crates/turbopack-ecmascript/tests/tree-shaker/analyzer/dce/output.md

```diff
@@ -80,27 +80,29 @@ graph TD
 # Final
```

Comment:
Original:

<img width="909" alt="image" src="https://github.com/user-attachments/assets/c0563d5c-5923-4477-983e-690796e3ea7d" />

New:

<img width="909" alt="image" src="https://github.com/user-attachments/assets/988a4691-077d-47ac-ab7e-498c8a8eb640" />

The `ImportBinding` is not merged because of static analysis like `next/dynamic`.

https://github.com/vercel/next.js/blob/494cac1fde7514734cd1f5ee5e6445d2b003e2e7/turbopack/crates/turbopack-ecmascript/src/tree_shake/optimizations.rs#L23-L44